spyder/app/tests/test_mainwindow.py

```diff
@@ -5430,7 +5430,9 @@ def test_clickable_ipython_tracebacks(main_window, qtbot, tmpdir):
                     timeout=SHELL_TIMEOUT)
 
     # Open test file
-    test_file = osp.join(LOCATION, 'script.py')
+    test_file_orig = osp.join(LOCATION, 'script.py')
+    test_file = osp.join(LOCATION, 'script_clickable.py')
+    shutil.copyfile(test_file_orig, test_file)
```

Comment:
```suggestion
    test_file = osp.join(str(tmp_path), 'script_clickable.py')
    shutil.copyfile(test_file_orig, test_file)

    # Open test file
```

* Copy script to a temp location to avoid polluting our repo when running this test.
* Please change the `tmpdir` fixture in this test (which was unused) to `tmp_path` (I can't leave a suggestion for that).